### ISSUE TITLE
[Suggestion]: Discord RPC

### ISSUE THREAD

**Issue by xIveilx** [dr3ams/Industrial-Village#65]
### General Info

- [ ] I would like to propose new mod
- [ ] I would like to propose new quests
- [ ] I would like to propose changes
- [X] I would like to propose another thing

### Suggestion Description

Hello, I'd love when i play this modpack and my friends see my activity on Dscord, they should see im playing this modpack. I hope you guys can add it to the modpack :3

**Comment by Gunner76th** [comment]
All mod suggestions must include a link to the mod on curseforge.

**Comment by Gunner76th** [comment]
Also for future reference, in the top section where you select the purpose of your post like this, you should select the top option, not the bottom one, as you are proposing a new mod.

**Comment by xIveilx** [comment]
okay, sorry  i didnt know it was a mod. :/

**Comment by Gunner76th** [comment]
It sounds like you are wanting something similar to Discord Rich Presence, or a similar mod.

**Comment by xIveilx** [comment]
> It sounds like you are wanting something similar to Discord Rich Presence, or a similar mod.

Yeah, like when i play the modpack the discord is show im playing Industrial Village etc.. 
And for shaders when you first launch the game your shader doesnt active firstly you need to turn it on to use it, but if i want to play with shaders with no problems, where can i download one?

**Comment by Gunner76th** [comment]
Curseforge is a good place to get shaders, as most are available on there. Or you can do a google search for them and download them directly from the mod authors website.

**Comment by Gunner76th** [comment]
that is a known issue with Immersive Engineering and shaders. The windmill, Waterwheel, and Excavator Bucket all do not render when you use shaders. There is no fix for the issue, as it is on their side (Immersive Engineering).

I am not sure if you are aware or not, but we do have a discord for general questions and conversation such as what this has turned into now. I suggest joining us on it:  https://discord.gg/r5N7PSgrDx
Then you can go to channel #industrial-village to discuss all things related to that modpack, or join one of the other modpack channels to discuss those ones.

**Comment by xIveilx** [comment]
okay thanks :D <3
